wire-ios-request-strategy/Sources/Request Strategies/Generic Message/GenericMessageEntity.swift

```diff
@@ -81,33 +81,32 @@ public func == (lhs: GenericMessageEntity, rhs: GenericMessageEntity) -> Bool {
 
 extension GenericMessageEntity: EncryptedPayloadGenerator {
 
-    public func encryptForTransport() -> EncryptedPayloadGenerator.Payload? {
+    public func encryptForTransport() async -> EncryptedPayloadGenerator.Payload? {
         guard
-            let conversation = conversation,
-            let managedObjectContext = conversation.managedObjectContext
+            let conversation = conversation
```

Comment:
We should probably move this inside the ` .conversationParticipants` case just we I did for the `encryptForTransportQualified()` otherwise it's impossible reach `encryptForTransport(forBroadcastRecipients:)`

Reply:
done